src/js/Controllers/DisplayCapabilities.js

```diff
@@ -1,287 +1,1689 @@
 let capabilities = {
 	"displayCapabilities": {
-		"displayType": "GEN2_8_DMA",
-		"displayName": "GENERIC_DISPLAY",
-		"textFields": [{
-				"name": "mainField1",
-				"characterSet": "TYPE2SET",
-				"width": 500,
-				"rows": 1
+		"MEDIA": {
```

Comment:
I feel like these should be structured like `capabilities.template.capabilityType` rather than `capabilities.capabilityType.template`.